lib/elixir/lib/kernel/lexical_tracker.ex

```diff
@@ -70,6 +84,10 @@ defmodule Kernel.LexicalTracker do
     :gen_server.cast(pid, {:remote_dispatch, module, mode})
   end
 
+  def remote_dispatch(pid, module, fas, line, mode) do
+    :gen_server.cast(pid, {:remote_dispatch, module, fas, line, mode})
```

Comment:
this should be called `fa` and not `fas` as it is a single function-arity and not a list of them.